src/components/ExportKeysForm.vue

```diff
@@ -119,77 +119,76 @@
 <script>
 import { validateMnemonic } from 'bip39'
 import copyToClipboard from 'copy-to-clipboard'
-
 import { getAccountFromPassphrase as getEthAccount } from '@/lib/eth-utils'
 import { getAccount as getBtcAccount } from '@/lib/bitcoin/btc-base-api'
 import { getAccount as getLskAccount } from '@/lib/lisk/lisk-api'
 import { Cryptos, CryptosNames } from '@/lib/constants'
 import QrcodeCapture from '@/components/QrcodeCapture'
 import QrcodeScannerDialog from '@/components/QrcodeScannerDialog'
-
-function getBtcKey (crypto, passphrase, asWif) {
-  const keyPair = getBtcAccount(crypto, passphrase).keyPair
-  const key = asWif
-    ? keyPair.toWIF()
-    : keyPair.privateKey.toString('hex')
-
-  return {
-    crypto: crypto,
-    cryptoName: CryptosNames[crypto],
-    key
-  }
-}
-
-function getLskKey (crypto, passphrase) {
-  const keyPair = getLskAccount(crypto, passphrase).keyPair
-  const key = keyPair.secretKey.toString('hex')
-
-  return {
-    crypto: crypto,
-    cryptoName: CryptosNames[crypto],
-    key
-  }
-}
+import { ref, reactive, computed } from 'vue'
+import { useStore } from 'vuex'
+import { useI18n } from 'vue-i18n'
 
 export default {
   components: {
     QrcodeCapture,
     QrcodeScannerDialog
   },
-  data () {
-    return {
-      passphrase: '',
-      showQrcodeScanner: false,
-      keys: []
+
+  setup () {
+    const passphrase = ref('')
+    const showQrcodeScanner = ref(false)
+    const store = useStore()
+    const { t } = useI18n()
+    let keys = reactive([])
+
+    const className = computed(() => {
+      return 'export-keys-form'
+    })
```

Comment:
Does not require to be reactive.

```suggestion
    const className = 'export-keys-form'
```